exist-core/src/test/resources/CompleteConf.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just wondering about the naming of this file. What is it that makes it `complete` compared to other configs? I realise it is used by your `ConfigurationTest.java` - but is there another reason why this is separate to the existing `conf.xml` used for test resources? or could you just add the xml-rpc settings to one of the existing files?

Reply:
> Just wondering about the naming of this file. What is it that makes it `complete` compared to other configs? I realise it is used by your `ConfigurationTest.java` - but is there another reason why this is separate to the existing `conf.xml` used for test resources? or could you just add the xml-rpc settings to one of the existing files?

@adamretter what would be the best file to refer instead of using a custom one?

Reply:
Can you explain what you mean by `complete` here? Then I can advise better...

Reply:
I tried to use  a better name